bucket/celestia.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "A real-time space simulation that lets you experience our universe in three dimensions.",
+    "homepage": "https://celestia.space/",
+    "version": "1.6.1",
+    "license": "GPL-2.0",
+    "url": "https://celestia.space/cc/celestia-win-161#celestia-161.exe",
+    "hash": "e7b9945e6aca5f5696d3c3be030963faef4c4991cd184222c7a4afbd72b5fd26",
+    "innosetup": true,
+    "bin": "celestia.exe",
```

Comment:
No shortcuts?

Reply:
Sorry, clicked merge just now ...
Will fix this :grin:

Reply:
i am... used to cli stuff. whoops.